src/index.js

```diff
@@ -227,26 +242,72 @@ export const EDSCTimeline = ({
     }
   }
 
+  /**
+   * Clear the current temporalRange value
+   */
   const clearTemporalRange = () => {
     setTemporalRange({})
 
     if (onTemporalSet) onTemporalSet({})
   }
 
-  const handleMoveTimeline = (newTimelinePosition) => {
+  /**
+   * Moves the timeline to the given position
+   * @param {Object} position Position to move the timeline to
+   */
+  const handleMoveTimeline = (position) => {
     if (onTimelineMove) {
-      const centeredDate = getCenterTimestamp({
+      const centeredTimestamp = getCenterTimestamp({
         intervalListWidthInPixels,
         timeIntervals,
-        timelinePosition: newTimelinePosition || timelinePosition,
+        timelinePosition: position || timelinePosition,
         timelineWrapperRef,
         zoomLevel
       })
 
-      onTimelineMove({ center: centeredDate, interval: zoomLevel })
+      onTimelineMove({ center: centeredTimestamp, interval: zoomLevel })
     }
   }
 
+  /**
+   * Move the timeline to the new center position
+   * @param {Integer} center Timestamp to move the timeline to
+   * @param {Array} intervals Optional - New timeIntervals to use
+   * @param {Integer} zoom Optional - New zoomLevel to use
+   */
+  const moveTimeline = (center, intervals = timeIntervals, zoom = zoomLevel, offset = 0) => {
```

Comment:
This could use a rename to something like `centerTimelineTo` or something, its a bit generic given its actual purpose.